clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AnnotationsBasedAtomicOperationsRegistry.groovy

```diff
@@ -57,27 +58,29 @@ class AnnotationsBasedAtomicOperationsRegistry extends ApplicationContextAtomicO
       value instanceof AtomicOperationConverter
     }.values().toList()
 
+    converters = converters.findAll { AtomicOperationConverter cv -> cv.acceptsVersion(version) }
```

Comment:
Why not pull this out into a helper function?

`converters = VersionHelper.findVersionMatches(converters)`
`validators = VersionHelper.findVersionMatches(validators)`

Or something injected if you don't want to use the static (to ease testing).